bazel/repositories.bzl

```diff
@@ -211,7 +218,7 @@ def _com_github_circonus_labs_libcircllhist():
     )
 
 def _com_github_c_ares_c_ares():
-    location = REPOSITORY_LOCATIONS["com_github_c_ares_c_ares"]
+    location = _strip_annotations(REPOSITORY_LOCATIONS["com_github_c_ares_c_ares"])
```

Comment:
Can you make this `_get_location(<the string>)` so that we can do other processing later and hide some of these implementation details?

Reply:
done

Reply:
done